connector/grafanacloudconnector/connector_test.go

```diff
@@ -60,25 +60,87 @@ func TestNewConnector(t *testing.T) {
 }
 
 func TestConsumeTraces(t *testing.T) {
+	factory := NewFactory()
 	testCases := []struct {
-		name   string
-		input  ptrace.Traces
-		output []pmetric.Metrics
+		name               string
+		cfg                *Config
+		input              ptrace.Traces
+		output             []pmetric.Metrics
+		expectedValue      string
+		expectedMetricsLen int
 	}{
 		{
-			name:  "sample",
-			input: testTraces(),
+			name: "default",
+			cfg:  factory.CreateDefaultConfig().(*Config),
+			input: testTraces(
+				map[string]string{
+					"host.id": "foo",
+				},
+			),
+			expectedValue:      "foo",
+			expectedMetricsLen: 1,
+		},
+		{
+			name: "multiple identifiers",
+			cfg: &Config{
+				HostIdentifiers: []string{"k8s.host.name", "host.id"},
```

Comment:
```suggestion
				HostIdentifiers: []string{"k8s.node.name", "host.id"},
```

Let's change this to `k8s.node.name` to match the semantic convention.

Reply:
We could also import from the `semconv` package for these, example: https://pkg.go.dev/go.opentelemetry.io/otel@v1.35.0/semconv/v1.30.0#K8SNodeName

Reply:
>We could also import from the semconv package for these, example: https://pkg.go.dev/go.opentelemetry.io/otel@v1.35.0/semconv/v1.30.0#K8SNodeName

In this case I think it's an overkill since the attribute is defined by the user anyway and using semconv requires some type conversions, WDYT?

Reply:
yeah good point! sounds good to me.